src/components/KCheckbox/KCheckbox.vue

```diff
@@ -59,4 +84,63 @@ export default defineComponent({
 @import '@/styles/variables';
 @import '@/styles/functions';
 
+.k-checkbox-label {
+  /* Body/Popover Content - Regular - 400 */
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  /* identical to box height, or 143% */
```

Comment:
nit: I don't think this comment applies to any of this CSS...?

Reply:
True, removed